be2-scala/docs/README.md

```diff
@@ -206,6 +206,8 @@ Summary of the keys used to retrieve data:
 - for a message: `channel#message_id`
 - for ChannelData: `channel`
 - for LaoData: `root/lao_id#laodata`
+- for RollCallData: `root/rollcall/lao_id`
```

Comment:
it's indexed by lao_id but not by roll_call_id ? 🤔

Disclaimer: this is not a documentation problem, just an odd choice that I'd forgotten about and possibly something the students will want to improve 😅